DiamondLightSource/python-artemis
# 8: Expand fgs system tests

Fixes #8 

Note, is branched from https://github.com/DiamondLightSource/python-artemis/pull/7 so merge that PR first.

To test:
* Ensure you have https://gitlab.diamond.ac.uk/controls/python3/s03_utils/-/merge_requests/4 running for the fast grid scan (this is not yet on ws019
* Run the system tests and confirm they all pass
